packages/core/src/Stores/Helpers/client-notifications.js

```diff
@@ -585,7 +594,13 @@ const checkAccountStatus = (
         } else if (ASK_CURRENCY) {
             addNotificationMessage(clientNotifications(ui_store).currency);
         } else if (ASK_AUTHENTICATE) {
-            addNotificationMessage(clientNotifications().authenticate);
+            const { has_poi, has_poa } = populateVerificationStatus(account_status);
```

Comment:
What happens with `.authenticate` notification key? Should we remove it?

Reply:
@reza-fs I just break it to two separate notifications as you can see. So we will not have a general authenticate notification instead we have these two
If you think that we need it too I can add a default to it

Reply:
@ali-hosseini-fs can we not remove `.authenticate` notification key? That's the approved content for high risk clients that need authentication. Also, can we use `needs_poi` and `needs_poa` notification keys instead of creating new notification keys?

Reply:
@farrah-deriv I don't think that we can show both of them at the same time, 
We have that authenticate notification but I just split it to two different notifications as POA and POI notifications
@mamali-fs  also I checked the conditions with @yashim-deriv and he suggested to add some other flags to make it stronger ( that you can see in the new changes ) and also we found a solution for rejected POI but we have a problem in term of POA that we don't have rejected status from BE for it. Currently i'm discussing it in the remove_mx_mlt channel